### DISCUSSION TITLE
May i add week 7 to my fnf engine?

### DISCUSSION THREAD

**Discussion by Jotaro-Gaming** [FunkinCrew/Funkin#1698] [General]
im making an engine and alot of people are asking me to put week 7 in it

**Comment by AbnormalPoof** [comment]
It's best you wait until Week 7 source code comes out
Though, PSXFunkin has Week 7 with permission from ninjamuffin. So maybe...

I'm not muffin so I can't really give you any advice

**Comment by ninjamuffin99** [comment]
no
PSXFunkin only has week 7 cuz its like an entirely different thing than just an FNF mod.
u can have week 7 if u port FNF to gameboy advance or some shit like that tho 👍👍👍👍

**Reply by AbnormalPoof** [reply]
what about virtual boy :troll:

**Reply by cuckydev** [reply]
what if u compile the psxfunkin engine for desktop like i did

**Reply by Jotaro-Gaming** [reply]
> no
> PSXFunkin only has week 7 cuz its like an entirely different thing than just an FNF mod.
> u can have week 7 if u port FNF to gameboy advance or some shit like that tho 👍👍👍👍

 oh ok

**Reply by steviegt6** [reply]
> 
> 
> what if u compile the psxfunkin engine for desktop like i did

How is this relevant?

**Reply by libauror-a** [reply]
ok im gonna port week 7 into geometry dash :trollface:

**Reply by GuglioREAL** [reply]
Someone ported the game to lovepotion are they allowed to include week 7?

**Comment by ninjamuffin99** [comment]
no

**Reply by AbnormalPoof** [reply]
should you mark your other comment as answered-
